tests/Provider/BuilderAliasProviderTest.php

```diff
@@ -5,10 +5,18 @@
 use Knp\Bundle\MenuBundle\Provider\BuilderAliasProvider;
 use Knp\Bundle\MenuBundle\Tests\Stubs\TestKernel;
 use PHPUnit\Framework\TestCase;
-use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
 
 class BuilderAliasProviderTest extends TestCase
 {
+    protected function setUp(): void
+    {
+        parent::setUp();
+        if (!interface_exists(ContainerAwareInterface::class)) {
```

Comment:
how is this related to the templating component ?

Reply:
shouldn't we skip only the test relying on a container-aware menu and not all tests for BuilderAliasProvider ?

And anyway, this should not be in a commit or PR saying it deprecates the templating integration if its fixes a different problem.

Reply:
You're right. I changed the skipping, now it skips only a single test